kong/pdk/private/node.lua

```diff
@@ -0,0 +1,86 @@
+local utils = require "kong.tools.utils"
+local pl_file = require "pl.file"
+local pl_path = require "pl.path"
+local pl_dir = require "pl.dir"
+
+local ngx = ngx
+
+local cached_node_id
+
+local function node_id_filename(prefix)
+  return pl_path.join(prefix, "kong.id")
```

Comment:
Avoid hard coded path names and ca we call it something like `.node_id` instead?